packages/shared/useFullscreen.ts

```diff
@@ -206,7 +206,7 @@ function useFullscreen(
   );
 
   return {
-    isEnabled: Boolean(element),
+    isEnabled: Boolean(document[fullscreenControls.fullscreenEnabled]),
```

Comment:
Is there a merit to doing this over the existing `element` variable? 🤔 

cc @simbathesailor

Reply:
Yes, as it is describe here : https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API#properties 

The element only tells you if fullscreen mode is activated (as `isFullscreen` is meant to)
```
Document.fullscreenElement / ShadowRoot.fullscreenElement
    The fullscreenElement property tells you the Element that's currently being displayed in full-screen mode on the DOM (or shadow DOM). If this is null, the document (or shadow DOM) is not in full-screen mode.
```

`fullscreenEnabled` tells you if fullscreen is supported or not
```
document.fullscreenEnabled
    The fullscreenEnabled property tells you whether or not it is possible to engage full-screen mode. This is false if full-screen mode is not available for any reason (such as the "fullscreen" feature not being allowed, or full-screen mode not being supported).
```

Reply:
Yes I agree with thodubois , document.fullscreenEnabled should be used instead of Document.fullscreenElement /  ShadowRoot.fullscreenElement. That's what I can understand from docs

Reply:
It seems it was like this before:
https://github.com/imbhargav5/rooks/blob/0180335c454c035c586246df10c7d9ad12f0275b/packages/shared/useFullscreen.ts#L150

Reply:
Sounds good!